styles.css

```diff
@@ -306,6 +485,104 @@ li {
     background-repeat: no-repeat;
     background-position: 100% 230px;
   }
+
+  .project_description {
+    width: 327px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #091e42;
+    padding-top: 12px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .project_image {
+    width: 95%;
+  }
+
+  .pop-header {
+    width: 100%;
+  }
+
+  .pop-header-h3 {
+    margin-top: 12px;
+    margin-bottom: 0;
+    font-weight: bold;
+    font-size: 32px;
+    color: #172b4d;
+    font-family: Poppins, sans-serif;
+    line-height: 44px;
+  }
+
+  .pop-header-div {
+    justify-content: flex-end;
+    float: right;
+    margin-top: -4%;
+    margin-left: 21%;
+  }
+
+  .pop-header-img {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .inf {
+    display: flex;
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    margin-top: 5px;
+    margin-left: 0;
+    color: #344563;
+  }
+
+  .mobile-part-class {
+    display: flex;
+    flex-direction: column-reverse;
+    margin-top: 20px;
+  }
+
+  .mobile-part-class img {
+    margin-top: 27%;
+  }
+
+  .pop-header-description {
+    width: 100%;
+    margin-right: 1%;
+    font-weight: 400;
+    font-size: 16px;
+  }
+
+  section#work {
+    padding-top: 10%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .techns {
+    display: flex;
+    align-items: baseline;
+    justify-content: flex-start;
+    padding-left: 0;
+    margin-top: 15%;
+    margin-bottom: 4%;
+  }
+}
+
+.pop-window {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  height: 90vh;
+  width: 95%;
+  margin-left: 2.5%;
+  padding: 3%;
+  margin-top: 10%;
```

Comment:
- [ ] I noticed that you made some changes requested, however, your popup design doesn't stick to the Figma design. This is due to the `margin-left: 55%` you gave to your `.pop-header-description` class and the `margin-top: 13%`, `margin-left: 32%`   you applied to the `.mobile-part-class div` on mobile version. Try to fix those values and your project will have a beautiful design 😊 
- [ ] OPTIONAL: To make your popup responsive, It would be better if you made some changes to the `height` value and `top` distance.
```suggestion
  height: 90%;
  width: 95%;
  margin-left: 2.5%;
  padding: 3%;
  margin-top: 2%;
```
**1. You design**
![image](https://user-images.githubusercontent.com/80612925/174129905-ac5737c0-24c0-474e-b0ad-b29abdcc49fc.png)
**2. Figma design**
![image](https://user-images.githubusercontent.com/80612925/174130476-13ac4207-0c2d-428d-9fbf-e528846bbb85.png)

Reply:
@Kandy-Peter Hi,

My desktop shows that is aligned, I can´t change it from there, please check it. Thanks
![Captura de pantalla (196)](https://user-images.githubusercontent.com/102225542/174152711-64bdc1e2-b303-49d5-aa59-33b7bb85a486.png)